main.yml

```diff
@@ -450,6 +450,23 @@ frontend-assets:
   deployment_repo: https://github.com/RedHatInsights/frontend-assets-build.git
   source_repo: https://github.com/RedHatInsights/frontend-assets
 
+image-builder:
+  title: Image Builder
+  api:
+    versions:
+      - v1
+  deployment_repo: https://github.com/RedHatInsights/image-builder-frontend-build
+  frontend:
```

Comment:
No need for this frontend config anymore, can you please however update the rhel navigation to include the image builder? https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/rhel-navigation.json#L60-L63 you will also have to add https://github.com/RedHatInsights/cloud-services-config/blob/ci-beta/chrome/fed-modules.json#L593-L604

Reply:
Since our GA date is may 2nd I thought to postpone the nav change a little bit longer. However for stage we can already enable it? And then production next week or so?